aten/src/ATen/DeviceGuard.h

```diff
@@ -14,7 +14,7 @@ namespace at {
 //    OptionalDeviceGuard guard(device_of(tensor));
 
 /// Return the Device of a Tensor, if the Tensor is defined.
-inline optional<Device> device_of(Tensor t) {
+inline optional<Device> device_of(const Tensor& t) {
```

Comment:
Should this change apply to other overloads of device_of?

Reply:
TensorList is an ArrayRef https://github.com/pytorch/pytorch/blob/693871ded3d3c643d7d57b990fc415198392b5f0/aten/src/ATen/templates/Tensor.h#L38 so I think no